usr/bin/sandbox-app-launcher

```diff
@@ -9,9 +9,11 @@ app_name="${1}"
 shift 1
 : ${app_path:="$(type -P "${app_name}" || true)"}
 
-if ! [[ "${app_name}" =~ [0-9a-zA-Z/] ]] || ! [[ "${app_path}" =~ [0-9a-zA-Z/] ]]; then
-  echo "ERROR: Invalid character."
-  exit 1
+if ! [[ "${app_name}" =~ [0-9a-zA-Z/] ]]; then
+  if ! [[ "${app_path}" =~ [0-9a-zA-Z/] ]]; then
```

Comment:
The should not be a nested `if`. No need. app_name could be OK while app_path is not.

Will add a commit on top to fix.